wiki/Performance_points/ru.md

```diff
@@ -1,166 +1,142 @@
 
-Высокоточные результаты считаются очень мастерскими алгоритмом очков производительности и будут награждаться очень большим счётом в сравнении с менее точными.
+Как следствие, на скорость карты применительно к PP сильно влияют моды [Double Time](/wiki/Game_modifier/Double_Time) и [Half Time](/wiki/Game_modifier/Half_Time).
 
-Так, например, результат, поставленный с точностью 80%, порою стоит 2/3 от результата с точностью 95%.
+### Точность
 
-Моды вроде Hidden, Hard Rock и Flashlight считаются существенно меняющими то, как сложно будет достигать точного результата на карте.
+*См. также: [Точность](/wiki/Gameplay/Accuracy)*
 
-### Что такое «нагрузка»?
+*Точность* (англ. *accuracy*) — процентный показатель того, насколько своевременно игрок попадает по объектам. Он также влияет на оценку производительности при расчёте PP.
 
-**Нагрузкой считается то, сколько раз и как долго игрок подвергается высокоинтенсивным секциям на определённой карте.**
+Чем выше точность рекорда, тем более значительным он считается и, как результат, тем больше PP принесёт: например, рекорды с [полным комбо (ФК)](/wiki/Full_combo) и точностью 80% могут приносить всего 2/3 PP от рекордов с точностью 95%. Поскольку алгоритм расчёта PP сильно завязан на точность, такие моды, как Hidden, Hard Rock и Flashlight значительно увеличивают число PP при условии высокой точности рекорда.
 
-Секции или резкие моменты чрезвычайно высокой скорости или сложности паттерна на карте будут значительно увеличивать подсчитанные значения нагрузки.
+### Нагрузка
 
-Карты с высоким значением нагрузки считаются очень и очень сложными, и следовательно награждают очень большими суммами очков производительности, если могут быть преодолены текущими навыками игрока.
+*Нагрузка* (англ. *strain*) — ключевой компонент, показывающий, сколько в карте секций с высокой интенсивностью. Также учитывается длительность таких секций.
 
-### Как аим, скорость, точность и нагрузка объединяются для создания pp?
+Нагрузку увеличивают участки карты, имеющие крайне высокую скорость или сложные [паттерны](/wiki/Beatmap/Pattern). В качестве примера можно привести карты с обилием [стримов](/wiki/Beatmap/Pattern/Stream) или секциями с быстрыми джампами — они принесут много PP.
 
-**Все четыре элемента согласованно рассматриваются для определения общей сложности карты, а также как конкретный результат сравнивается с тем, что считается «оптимальной» игрой для этой карты.**
+<!-- consult reg contributors before moving FAQ to a new location-->
 
-Алгоритм для очков производительности существенно варьируется в зависимости от режима игры.
+## FAQ
 
-Хотя точные цифры выходят далеко за рамки данной статьи, некоторые игровые режимы имеют больший приоритет над определенной статистикой из-за их индивидуальной механики.
+### Где найти ранкинг по PP?
 
-### Что за «система взвешивания» и как она влияет на подсчёт моих результатов?
+**Здесь: [ранкинг игроков по производительности](https://osu.ppy.sh/p/pp).**
```

Comment:
missing line?

```md
### Where can I view the performance ranking?

**The performance points ranking for all players can be found on the [rankings page](https://osu.ppy.sh/p/pp).**

You can also navigate to the rankings by using the `ranking` dropdown panel at the top of the legacy web design, and choosing the `performance` option.
```

also i wonder how the hell did this link not get replaced with a new site route 🤨

Reply:
I don't see any point in referring to the legacy website when it's long phased out